lib/message_queue/adapters/rabbitmq/producer_worker.ex

```diff
@@ -35,6 +35,7 @@ defmodule MessageQueue.Adapters.RabbitMQ.ProducerWorker do
       ProcessRegistry.register(:producer_workers, chan)
       Process.monitor(conn.pid)
       Process.monitor(chan.pid)
+      Logger.warning("[ProducerWorker] started #{inspect(conn.pid)} #{inspect(chan.pid)}")
```

Comment:
maybe use `chan`, it will have all info about `chan` and `conn`
```suggestion
      Logger.info("[ProducerWorker] started #{inspect(chan)}")
```